packages/beacon-node/src/chain/shufflingCache.ts

```diff
@@ -31,6 +36,7 @@ type ShufflingCacheItem = {
 
 type PromiseCacheItem = {
   type: CacheItemType.promise;
+  timeInserted: number;
```

Comment:
propose to have suffix as "Ms" or "Sec" to make it more straightforward in unit, for example we have `seenTimestampSec` variable